web/src/components/CityContentHeader.tsx

```diff
@@ -64,31 +61,27 @@ const CityContentHeader = ({
 
   const { t } = useTranslation('layout')
 
+  const SearchButton = (
+    <HeaderActionBarItemLink
```

Comment:
When navigating to the search I see that the search icon in the search directly is also inverted when changing language. So I think we should either also invert the header icon or not revert the search icon next to the search query:
<img width="831" alt="Bildschirm­foto 2023-03-16 um 11 34 12" src="https://user-images.githubusercontent.com/28013368/225591578-516dbcfe-a47f-4ed9-ae4f-2be9109de6f8.png">

Reply:
Nice catch, we are already doing that in main but that's very much not in the design. I'll remove the inversion of the search symbol.